quinn-proto/src/tests.rs

```diff
@@ -88,6 +88,7 @@ fn server_config() -> Config {
     };
 
     let mut tls_server_config = crypto::build_server_config();
+    tls_server_config.set_protocols(&["hq-11".into()]);
```

Comment:
I think we probably still want `"hq-11"` defined somewhere as a constant instead of writing it as a literal all over the place. Maybe a constant named `ALPN_QUIC_HTTP` in `quinn-proto/src/lib.rs`?

Reply:
Ideally this constant would live in a separate HTTP-over-QUIC crate, but this'll do for the short term.

Reply:
The usages in `quinn-proto/src/tests.rs` should also use the constant. Can you update your second commit, please?